CHANGELOG.asciidoc

```diff
@@ -3,6 +3,104 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.10.0]]
+=== Beats version 8.10.0
+https://github.com/elastic/beats/compare/v8.9.2\...v8.10.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+- Fix status reporting to {agent} when output configuration is invalid running under Elastic-Agent {pull}35719[35719]
+- Upgrade Go to 1.20.7 {pull}36241[36241]
```

Comment:
These aren't breaking changes. The first is a bug fix, the second can go under the Added section.